clusters/build-clusters/02_cluster/openshift-machine-api/pthread-debug_mcp.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
```

Comment:
Doesn't the MachienSet also need to specify its machineconfigpool?

Reply:
https://github.com/openshift/machine-config-operator/blob/master/docs/custom-pools.md
I do not see any machineset mentioned ^^^.

This is my understanding:
MachineSet is a concept of `openshift-machine-api`. It provisions machines/nodes using cloud API.
MachineConfig and MachineConfigPool are from `openshift-machine-config-operator`. They work on nodes level (see the nodeSelector in its spec).

This is from [the MCP doc](https://github.com/openshift/machine-config-operator/blob/master/docs/custom-pools.md):
_Since a custom pool inherits from the worker pool, any change to the worker pool is going to roll out to the custom pool as well (like an OS update during an upgrade)._